chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -203,6 +204,17 @@ def usage
           self.owner
         end
 
+        def multipath?(device = @device)
+          uuid_path = "/sys/block/#{device}/dm/uuid"
+          File.exist?(uuid_path) && File.open(uuid_path) { |f| f.read(7).start_with?("mpath-") }
+        end
+
+        def multipath_path?
```

Comment:
As requested by Itxaka, who is now working on it. Given:
```
root@d52-54-77-77-01-01:~ # multipath -ll
0QEMU_QEMU_HARDDISK_00002 dm-1 QEMU,QEMU HARDDISK
size=10G features='0' hwhandler='0' wp=rw
|-+- policy='service-time 0' prio=1 status=active
| `- 0:0:0:2 sdc 8:32   active ready running
`-+- policy='service-time 0' prio=1 status=enabled
  `- 0:0:0:3 sdb 8:16   active ready running
0QEMU_QEMU_HARDDISK_00001 dm-0 QEMU,QEMU HARDDISK
size=10G features='0' hwhandler='0' wp=rw
|-+- policy='service-time 0' prio=1 status=active
| `- 0:0:0:0 sda 8:0    active ready running
`-+- policy='service-time 0' prio=1 status=enabled
  `- 0:0:0:1 sdd 8:48   active ready running
```
`sdb` and `sdc` are paths of `dm-1`.

```
root@d52-54-77-77-01-01:~ # ls /sys/block/sdb/holders/                                                                                                                                  
dm-1
root@d52-54-77-77-01-01:~ # ls /sys/block/sda/holders/
dm-0
root@d52-54-77-77-01-01:~ # ls /sys/block/sdb/holders/
dm-1
root@d52-54-77-77-01-01:~ # ls /sys/block/sdc/holders/
dm-1
root@d52-54-77-77-01-01:~ # ls /sys/block/sdd/holders/
dm-0
```